harness/features/bootstrapping.test.ts

```diff
@@ -11,12 +10,7 @@ describe('Bootstrapping Tests', () => {
 
         describe('not enabled client', () => {
             beforeEach(async () => {
-                testClient = new LocalTestClient(sdkName)
-                scope
```

Comment:
why is this removed?

Reply:
I'm not sure, but the test didn't break? Might be from my rebase?

Reply:
Oh, it was from the rebase, it is lower down

Reply:
The scope param was passed in - so it sets it up in the constructor

Reply:
what constructor? like this this is the code on main right now, is it not setting up/testing for the config mock anymore?